application/views/questionAdministration/subquestionRow.twig

```diff
@@ -129,18 +129,21 @@
         </div>
     </td>
 
+    {% set relevanceTooltip = supportsRelevance ? gT("Click to expand") : gT("Relevance is not currently supported for columns") %}
+
     <!-- Relevance equation -->
     {% if first and scale_id == 0 %}
         <td class="relevance-equation">
             <input
                 data-toggle="tooltip"
-                data-title="{{ gT("Click to expand") }}"
+                data-title="{{ relevanceTooltip }}"
                 type='text'
                 class='relevance form-control input'
```

Comment:
Why not hide ?

Reply:
@c-schmitz Asked for it like this (if I understood correctly)

Reply:
@c-schmitz  write

> Either introduce a question attribute that **hides the relevance column** in the subquestion editor for this question type (and other question types that don't use the relevance properly)

https://bugs.limesurvey.org/view.php?id=17067#c71103

Reply:
Hiding is a way to say, "not allow people to use it". 
Making readonly is similar and I believe clearer, easier and less confusing.
If someone was expecting to use ut and suddnely is hidden, they will be confused.

Reply:
Great to see this. I think it would help the user (developer of surveys) if a tooltip is added: "Conditionally hiding columns is not (yet) possible, due to technical constraints."

If no help is given, people will look into the manual, ask questions in the forum and add bug reports because this options is greyed out.

Reply:
And talking about relevance has been changed to talking about conditions, hasn't it?

Reply:
Adding a specific class in td (and in th) then …

@c-schmitz ask to hide column , and we already hide column for Y scale

![Capture d’écran du 2022-07-29 15-56-57](https://user-images.githubusercontent.com/1439428/181775932-ba638ee6-4e72-411d-9a93-38c0e1f2235c.png)